app/core/tests/gae_suite.py

```diff
@@ -35,9 +35,11 @@
 OPPIA_TOOLS_DIR = os.path.join(CURR_DIR, '..', 'oppia_tools')
 
 DIRS_TO_ADD_TO_SYS_PATH = [
-    CURR_DIR,
+    CORE_DIR,
```

Comment:
Why was this changed? I notice a "from core.tests" in the file above and that seems like how it should be...

Reply:
Yep I do not need to make any changes here!